### ISSUE TITLE
handle unmapped reads in BAM

### ISSUE THREAD

**Issue by tseemann** [andersen-lab/ivar#8]
Would be great if unmapped reads could be left alone and ignored?

```
% ivar trim -i ivar.bam
Segmentation fault
```

```
# remove unmapped reads
% samtools view -F 4 -o ivar.bam ivar2.bam
% ivar trim -i ivar2.bam
# works!
```

**Comment by gkarthik** [comment]
Unmapped reads should just not be written to the trimmed BAM file. There shouldn't be a segmentation fault. Would it be possible for you to provide a test file for which you got the seg fault? Thank you! 

I'll include an option to write unmapped reads to the trimmed file.

**Comment by gkarthik** [comment]
Fixed by 1b426123c560197db3c9ee44174b81538cdb6383
